src/lib/common/limits.h

```diff
@@ -148,6 +148,13 @@
 
 
 
+/* ****************************************************************************
+*
+* MAX_SUB_HTTP_TIMEOUT - 300000 milliseconds, 5 minutes
+*/
+#define MAX_SUB_HTTP_TIMEOUT     300000
```

Comment:
Maybe the same value used by `-httpTimeout` (in the CLI) should be used? From https://github.com/telefonicaid/fiware-orion/blob/master/src/app/contextBroker/contextBroker.cpp#L315 it seems it is unbounded (i.e. as long a the containing type `long` can be).

Or, the other way around, use for `-httpTimeout` the same limit we are using here.

Reply:
> Or, the other way around, use for `-httpTimeout` the same limit we are using here.

I think this makes more sense. IMHO the reason of using this parameter just for one notification is to have a bigger timeout compared to the default one. Another option is to define the maximum using an envar.

Reply:
Ok, let's go for it. Then you have to use this MAX_SUB_HTTP_TIMEOUT at https://github.com/telefonicaid/fiware-orion/blob/master/src/app/contextBroker/contextBroker.cpp#L315 instead of the current UINT_MAX.

Not sure if 5 minutes is a good hard limit (taking into account that now is more or less "infinite" :). Maybe we could enlarge a bit that? 30 minutes?

Whichever the time, it should be also annotated in the CNR file, e.g:

```
- Add: hard limit for -requestTimeout to xxxxxx millliseconds
```